### PR TITLE
Improve sliding menu

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

File: components/SlidingMenu/index.vue [hunk 1]
```diff
@@ -1,74 +1,109 @@
...
+          </div>
+
+          <div class="social-media-links-wrapper">
+            <platform-icon-bar />
+          </div>
+
+          <div class="menus-wrapper">
+            <div
+              v-for="m in MENU_ITEMS"
+              :class="['menu', m.section]">
```
Comment (williamchong007) [inline components/SlidingMenu/index.vue]: 呢個 for loop 會唔會被鬧無 `:key` ？ 定係有下面果個就 ok？

File: components/SlidingMenu/index.vue [hunk 2]
```diff
@@ -1,74 +1,109 @@
...
+import PlatformIconBar from '~/components/PlatformIconBar';
+import MenuItem from './MenuItem';
+
+const MENU_ITEMS = [
+  {
+    section: 'primary',
+    items: [
+      {
+        key: 'aboutLikeCoin',
+        to: '/',
```
Comment (williamchong007) [inline components/SlidingMenu/index.vue]: why not `to { name: 'index' }`?

File: components/SlidingMenu/index.vue (hunk 1 above)
Comment (nwingt) [reply components/SlidingMenu/index.vue]: 佢無鬧我無key 我估可能我改咗class 不過我照加番key

Comment (nwingt) [reply components/SlidingMenu/index.vue]: @williamchong007 Updated

File: components/SlidingMenu/index.vue (hunk 2 above)
Comment (nwingt) [reply components/SlidingMenu/index.vue]: Updated

**Pull request merged**